controllers/hetznerbaremetalhost_controller.go

```diff
@@ -168,10 +168,6 @@ func (r *HetznerBareMetalHostReconciler) Reconcile(ctx context.Context, req ctrl
 		}
 	}
 
-	log = log.WithValues("HetznerBareMetalMachine", klog.KObj(hetznerBareMetalMachine))
```

Comment:
here the change still exists

Reply:
@guettli the change still exists. Can you get it back to the original state with regards to the logger? Except of course the additional value that you write into the loggers values.

Reply:
I reverted these changes concerning WithValues, and created a new PR: [:seedling: log less values in hbmh Reconcile. by guettli · Pull Request #1606 · syself/cluster-api-provider-hetzner](https://github.com/syself/cluster-api-provider-hetzner/pull/1606)